hat/assets/js/apps/Iaso/routing/redirections.js

```diff
@@ -63,6 +64,14 @@ const addRoutes = (baseRoutes, overrideLanding) => {
                 storageDefaultSort,
             )}/pageSize/20/page/1`}
         />,
+        <Redirect
+            path={`${baseUrls.workflows}/entityTypeId/:entityTypeId`}
+            to={`${
+                baseUrls.workflows
+            }/entityTypeId/:entityTypeId/order/${getSort(
+                workflowDefaultSort,
+            )}/pageSize/20/page/1`}
+        />,
```

Comment:
probably need to add accountId here as well

Reply:
This change requires the rest of the accountId PR to work. Changing now will break the page

Reply:
💥